content/docs/concepts/clusters.mdx

```diff
@@ -0,0 +1,185 @@
+1. Under **Actions**, select the **edit** icon
+1. Select the **Domains** tab
+1. In the **Cluster Identity** field, select **Rotate Identity**
+1. Copy the token value and store it somewhere safe ![Rotating a cluster identity in the Zero Console](./img/clusters/pz-rotate-cluster-identity.gif)
+
+After rotating the cluster identity, you must pass it into your configuration and run Pomerium. See the steps below for your deployment environment.
+
+<Tabs>
+<TabItem value="linux" label="Linux">
+
+In the shell script below, replace `<cluster_token>` with your new cluster token before running the command:
+
+```bash
+curl https://console.pomerium.app/install.bash | env POMERIUM_ZERO_TOKEN=<cluster_token> bash -s install
+```
+
+</TabItem>
+<TabItem value="docker" label="Docker">
+
+1. In your `docker-compose.yaml` file, replace the value of the `POMERIUM_ZERO_TOKEN` environment variable with the new cluster token value:
+
+   ```yaml title="docker-compose.yaml"
+   services:
+     pomerium:
+       image: pomerium/pomerium:v0.25.1
+       ports:
+         - 80:80
+         - 443:443
+       restart: always
+       environment:
+       // highlight-start
+         POMERIUM_ZERO_TOKEN: <CLUSTER_TOKEN>
+       // highlight-end
+         XDG_CACHE_HOME: /var/cache
+         AUTOCERT_DIR: /var/cache/autocert
+       volumes:
+         - pomerium-cache:/var/cache
+
+   volumes:
+     pomerium-cache:
+   ```
+
+1. Run `docker compose up -d`
+
+</TabItem>
+<TabItem value="kubernetes" label="Kubernetes">
+
+1. Replace the value of `pomerium_zero_token` with your new cluster token:
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: pomerium
+     namespace: pomerium-zero
+   type: Opaque
+   stringData:
+     pomerium_zero_token: <cluster_token>
+   ```
+1. Apply the manifest to your Kubernetes cluster
+   ```bash
+   kubectl apply -f ./secret.yaml
+   ```
+
+</TabItem>
+</Tabs>
+
+### Cluster domain
+
+Pomerium Zero generates and assigns a unique **Cluster Domain** to your cluster. A cluster domain contains a randomly generated subdomain and the `pomerium.app` domain.
+
+It follows this format: `<CLUSTER-SUBDOMAIN-XXXX>.pomerium.app`
+
+For example: `voracious-ape-1578.pomerium.app`
+
+#### Cluster name
+
+The **Cluster Name** is a customizable identifier for your cluster. It defaults to the randomly generated subdomain of your cluster domain, but you can change it at any time. Changing the cluster name does not affect your cluster domain.
+
+If you want to change the cluster name, you can update it in your cluster settings:
+
+1. From the navigation bar dropdown, select **Manage Clusters**
+1. Under **Actions**, select the **edit** icon
+1. In the **Cluster Name** field, enter your preferred cluster name
+1. Save your changes and apply the changeset
+
+![Viewing the new cluster name in the Zero Console](./img/clusters/cluster-name-table.png)
+
+#### Detected and Override IP address
```

Comment:
I personally think we should include this section here. At a minimum, we should cover these steps in Troubleshooting, or cross-link these pages.